Radio_continuum_validation.py

```diff
@@ -67,31 +70,26 @@
     warnings.warn_explicit("""Use option -h to see usage.\n""",UserWarning,WARN,cf.f_lineno)
     sys.exit()
 
-#don't use normal display environment unless user wants to view plots on screen
+# don't use normal display environment unless user wants to view plots on screen
 import matplotlib as mpl
```

Comment:
I think you should also move this to the top of the file

Reply:
Done.